database/v1/index.js

```diff
@@ -32,4 +32,8 @@ Object.keys(db).forEach(modelName => {
 db.sequelize = sequelize;
 db.Sequelize = Sequelize;
 
+db.sequelize.sync({ force: false }).then(data => {
```

Comment:
Are we still using v1 database? If we don't pls remove it

Reply:
Let's remove this DB in the next iteration.